cookies/domain/support/idn-child.sub.https.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <script src="/resources/testharness.js"></script>
+  <script src="/resources/testdriver.js"></script>
+  <script src="/resources/testdriver-vendor.js"></script>
+  <script src="/cookies/resources/cookie-test.js"></script>
+</head>
+<body>
+  <script>
+    async function assert_cookie(response) {
+      const get = await response.text();
+      assert_equals(get, "a=b");
```

Comment:
Consider naming your cookies something more unique. We've had flaky tests in the past due to stale cookie jars and while that situation is improving this seems like an easy way to be more confident in your test results.

Reply:
Good idea :)